tdrs-backend/tdpservice/parsers/models.py

```diff
@@ -86,47 +86,22 @@ class Status(models.TextChoices):
 
     datafile = models.ForeignKey(DataFile, on_delete=models.CASCADE)
 
-    #  eventually needs a breakdown of cases (accepted, rejected, total) per month
-    #  elif qtr2 jan-mar
-    #  elif qtr3 apr-jun
-    #  elif qtr4 jul-sept
-
     case_aggregates = models.JSONField(null=True, blank=False)
```

Comment:
i've only come across this during some manual testing (not yet during an actual upload/parse), but theoretically attributes in a JSON object are unordered. I expect it to work more like a text field in the database/backend, but by the time it's parsed into a javascript object on the frontend, the order of the months might not be preserved.